runtime/lua/vim/ui/float.lua

```diff
@@ -0,0 +1,214 @@
+local Border = require('vim.ui.border')
+local tbl = require('vim.ui.ui_table')
+
+_AssociatedBufs = {}
```

Comment:
This should be some other name I think. Like `__NvimUI_AssociatedBufs` or something. We don't want to stomp on anyone's globals.